src/replit_river/codegen/client.py

```diff
@@ -763,26 +762,25 @@ def generate_individual_service(
     schema: RiverService,
     input_base_class: Literal["TypedDict"] | Literal["BaseModel"],
 ) -> Tuple[ModuleName, ClassName, dict[RenderedPath, FileContents]]:
+    serdes: list[Tuple[list[TypeName], list[ModuleName], list[FileContents]]] = []
+
     def append_type_adapter_definition(
         type_adapter_name: TypeName,
         _type: TypeExpression,
         module_info: list[ModuleName],
     ) -> None:
         rendered_type_expr = render_type_expr(_type)
+        var_name = render_type_expr(type_adapter_name)
+        var_type = f"TypeAdapter[{rendered_type_expr}]"
+        var_value = f"TypeAdapter({rendered_type_expr})"
         serdes.append(
             (
                 [type_adapter_name],
                 module_info,
-                [
-                    FileContents(
-                        f"{type_adapter_name.value}: TypeAdapter[{rendered_type_expr}] = "
-                        f"TypeAdapter({rendered_type_expr})"
-                    )
-                ],
+                [FileContents(f"{var_name}: {var_type} = {var_value}")],
```

Comment:
this has a similar problem as before -- please maintain as much structure as possible to make this readable. Very hard to reason about `{a}: {b} = {c}`, much easier to reason about `{a}: TypeAdapter[{b}] = TypeAdapter({c})`

Reply:
this is what `textwrap.dedent` is for, it just strips the beginning whitespace from a whole line:

```suggestion
                [FileContents(dedent(f"""\
    {render_type_expr(type_adapter_name)}: TypeAdapter[{rendered_type_expr}] = TypeAdapter({rendered_type_expr})
                  """))],
```

you can play with it however you'd like -- the intent here is that the rendered form is similar in structure to the generated code to make associating and debugging easier

Reply:
I prefer this to render just 1 line of code, and dedent implies multiline.